docs/concepts/services-networking/service.md

```diff
@@ -453,7 +453,8 @@ cloud provider does not support the feature, the field will be ignored.
 
 **Special notes for Azure**: To use user-specified public type `loadBalancerIP`, a static type
 public IP address resource needs to be created first, and it should be in the same resource
-group of the cluster. Then you could specify the assigned IP address as `loadBalancerIP`.
+group of the cluster. Then you could specify the assigned IP address as `loadBalancerIP`. You 
```

Comment:
Can this be updated to show an example of how `loadBalancerIP` and `securityGroupName` should be used? The wording as it is highlights that there's things needed, but it's very unclear how to use it or if it's specifically required.

As a suggestion, could you reword this using more imperative, direct english:
> Specify the assigned IP address as `loadBalancerIP`. Verify you have `securityGroupName` in the cloud provider configuration file.

Or something like that?